src/types/json.h

```diff
@@ -215,6 +215,30 @@ struct JsonValue {
     return Status::OK();
   }
 
+  StatusOr<size_t> Del(const std::string &path) {
+    const std::string parent_path = path + "^";
+    const std::string child_path = path.substr(path.find_last_of('.') + 1);
```

Comment:
It is not a general way to retrieve the key, e.g. it can be wrong while processing `$[1]`, `$["a"]`, `$.a[*]`.

TBH it is hard to retrieve the key here, but you can try to cover more cases.

Reply:
Refer to https://github.com/danielaparker/jsoncons/issues/467#issuecomment-1784002194.

Reply:
According to https://github.com/danielaparker/jsoncons/issues/467#issuecomment-1784123066, it seems that `jsoncons` has not such feature. 😭